### PR TITLE
Add snapshot timestamp to cluster labels and use it to select deploying cluster

### LINKED ISSUE #295: Cronjob worker doesn't have permission to delete deployment
Crojob worker is not able to delete deployment with command:
gcloud deployment-manager deployments delete *

Error returned was: Deletion was aborted by user.

### PR DESCRIPTION
**Pull request by gabrielwen**
### Changes in this PR:
- Add snapshot timestamp to cluster labels.
- Use labels attached to select cluster to deploy.
- Add delete_deployment.sh to delete deployment before re-create it.

Fixes #295 

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/testing/308)

### CODE REVIEW

**Comment by gabrielwen** [comment]
Fixes #295

**Comment by gabrielwen** [comment]
Relates #95

File: py/kubeflow/testing/create_kf_instance.py [hunk 1]
```diff
@@ -62,6 +62,14 @@ def main(): # pylint: disable=too-many-locals,too-many-statements
     "--cluster_num",
     default="", type=int, help=("Number of cluster to deploy to."))
 
+  parser.add_argument(
```
Comment (jlewi) [inline py/kubeflow/testing/create_kf_instance.py]: Why are you passing this as an argument? Why don't you just have the code compute the timestamp?

Comment (gabrielwen) [reply py/kubeflow/testing/create_kf_instance.py]: this timestamp is the time deployed codes taken snapshot, not the time it is deployed.

**Pull request merged**